### ISSUE TITLE
aloha arm do not move in sim eval env on my own task

### ISSUE THREAD

**Issue by cong1024** [huggingface/lerobot#320]
### System Info

```Shell
ubuntu 20 , ros1
```

### Information

- [ ] One of the scripts in the examples/ folder of LeRobot
- [ ] My own task or dataset (give details below)

### Reproduction

Hi guys, i met a problem in sim env evaluation on my own task, there may be the bug or my own mistake.
my own task is just touch the box1 and then touch the box2, 

https://github.com/user-attachments/assets/16a12a49-9d70-4797-872a-b628bd0d9145

I change the sim env of original insertion task to my own task.  After training with lerobot framework, I got this:

https://github.com/user-attachments/assets/eefe7938-f44a-4bd0-a3bb-3ccefd70025e

Can you help me figure out the problem why the policy trained so badly? thanks!

### Expected behavior

not yet

**Comment by Cadene** [comment]
To first debug, you need to try to replay a training episode in your environment.
We dont have code for that yet, but it should be easy to do.
You can evaluate the policy using the same code as you already do, but instead of running the policy, you get actions from your dataset.
You need to be able to replay the episode.

Then you need to look for possible mismatch in the input and output of your policy compared to your training set.

Then you need to make sure that your training loss is very low.

**Comment by fgc346** [comment]
@cong1024 ，I am  curious that how your own simulator data generate？
